docs/quickstart.asciidoc

```diff
@@ -101,77 +108,100 @@ spec:
 
 CAUTION: Setting `node.store.allow_mmap: false` has performance implications and should be tuned for production workloads as described in the <<{p}-virtual-memory>> section.
 
+NOTE: If your Kubernetes cluster does not have any Kubernetes nodes with at least 2GiB of free memory, the pod will be stuck in `Pending` state. Check <<{p}-managing-compute-resources>> for more information about resource requirements and how to configure them.
+
+NOTE: The cluster that you deployed in this quickstart guide only allocates a persistent volume of 1GiB for storage using the default link:https://kubernetes.io/docs/concepts/storage/storage-classes/[storage class] defined for the Kubernetes cluster. You will most likely want to have more control over this for production workloads. Refer to <<{p}-volume-claim-templates>> for more information.
+
+For a full description of each `CustomResourceDefinition` (CRD), refer to the <<{p}-api-reference>> or view the CRD files in the link:{eck_github}/tree/{eck_release_branch}/config/crds[project repository]. You can also retrieve information about a CRD from the cluster. For example, describe the {es} CRD specification via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_describe/[`describe`]:
+
+[source,sh]
+----
+kubectl describe crd elasticsearch
+----
+
 [float]
 [id="{p}-elasticsearch-monitor-cluster-health"]
 === Monitor cluster health and creation progress
 
-Get an overview of the current Elasticsearch clusters in the Kubernetes cluster, including health, version and number of nodes:
+Get an overview of the current {es} clusters in the Kubernetes cluster via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`], including health, version and number of nodes:
 
 [source,sh]
 ----
 kubectl get elasticsearch
 ----
 
+When you first create the Kubernetes cluster, there is no `HEALTH` status and the `PHASE` is empty. After the pod and service start-up, the `PHASE` turns into `Ready`, and `HEALTH` becomes `green`. The `HEALTH` status comes from {es}'s link:{ref}/cluster-health.html[cluster health API].
+
 [source,sh,subs="attributes"]
 ----
 NAME          HEALTH    NODES     VERSION   PHASE         AGE
-quickstart    green     1         {version}     Ready         1m
+quickstart              1         {version}               1s
 ----
 
-When you create the cluster, there is no `HEALTH` status and the `PHASE` is empty. After a while, the `PHASE` turns into `Ready`, and `HEALTH` becomes `green`. The `HEALTH` status comes from link:{ref}/cluster-health.html[Elasticsearch's cluster health API].
-
-One Pod is in the process of being started:
+While the {es} pod is in the process of being started it will report `Pending` as seen via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`]:
```

Comment:
```suggestion
While the {es} pod is in the process of being started it will report `Pending` as seen with link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`]:
```